README.md

```diff
@@ -8,132 +8,192 @@ The Dark Sky API returns a wealth of data (in JSON format). As such, it is left
+#### Parameters ####
 
-If you choose not to provide a callback, the forecast will be made synchronously (blocking) and a table containing *err* and *data*, as above, will be returned by *forecastRequest()*. If the request is made asynchronously, *forecastRequest()* does not return anything.
+| Parameter | Data Type | Required | Description |
+| --- | --- | --- | --- |
+| *longitude* | Integer, float or string | Yes | The longitude of the location for which you’d like a forecast |
+| *latitude* | Integer, float or string | Yes | The latitude of the location for which you’d like a forecast |
+| *callback* | Function | No | Function called when the forecast has been received (or an error occurred) |
+
+If you provide a callback function, the request will be made asynchronously and the callback executed with the returned data. Your callback function must include the following parameters:
+
+| Parameter | Data Type | Description |
+| --- | --- | --- |
+| *err* | String | A human-readable error message error message, or `null` if there was no error |
+| *data* | Table | The decoded response from Dark Sky |
 
 The data returned by the Dark Sky API is complex and is not parsed in any way by the library. However, *data* contains an additional key, *callCount*, which is the number of calls you have made to the Dark Sky API. This is decoded by the library and added to the returned data table.
 
-#### Example
+#### Return Value ####
+
+Table (the decoded response from Dark Sky) or nothing (if *forecastRequest()* is performed asynchronously).
+
+#### Example ####
 
 ```squirrel
 fc.forecastRequest(myLongitude, myLatitude, function(err, data) {
-    if (err) server.error(err);
-
-    if (data) {
-        server.log("Weather forecast data received from Dark Sky");
-        if ("hourly" in data) {
-            if ("data" in data.hourly) {
-                // Get second item in array: this is the weather one hour from now
-                local item = data.hourly.data[1];
-                local sendData = {};
-                sendData.cast <- item.icon;
-                sendData.temp <- item.apparentTemperature;
-                device.send("weather.show.forecast", sendData);
-
-                // Log the output
-                local celsius = ((sendData.temp.tofloat() - 32.0) * 5.0) / 9.0;
-                local message = "Outlook: " + sendData.cast + ". Temperature: " + format("%.1f", celsius) + "ºC";
-                server.log(message);
-            }
-        }
-
-        if ("callCount" in data) server.log("Current Dark Sky API call tally: " + data.callCount + "/1000");
+  if (err) server.error(err);
+
+  if (data) {
+    server.log("Weather forecast data received from Dark Sky");
```

Comment:
https://developer.electricimp.com/squirrel/codestyle:

Indentations should never be tabs (\t) and should always be four spaces.

Reply:
I thought they were. Old-ish code, so I'll clean out the tabs.